third_party/blink/renderer/core/cobalt/performance/performance_extensions.idl

```diff
@@ -2,6 +2,9 @@
 [
     ImplementedAs=PerformanceExtensions
 ] partial interface Performance {
+    // Returns the application startup duration (us) in bytes, according to
+    // base::SysInfo::AppStartupTime().
+    [CallWith=ScriptState] unsigned long long getAppStartupTime();
```

Comment:
Unless absolutely needed, we should try to use asynchronous Promises (https://www.w3.org/TR/design-principles/#one-time-events). If this API is kept for legacy compatibility reasons, ideally we'd quote that here as e.g. H5vccMetrics does in https://source.corp.google.com/h/lbshell-internal/cobalt_src/+/main:third_party/blink/renderer/modules/cobalt/h5vcc_metrics/h_5_vcc_metrics.idl;l=15-17?q=h5vcc_metrics.idl&ss=h%2Flbshell-internal%2Fcobalt_src%2F%2B%2Frefs%2Fheads%2Fmain -- @hlwarriner and @andrewsavage1 FYI.

Reply:
I think we can do async here, but let me check with Sergei

Reply:
Talked to Sergei and Niketh and they're fine with it being async so we should make it async. @briantting lmk if you need me to walk you through how to do that, but there are a number of h5vcc examples already for you to reference for the implementation